core/src/hash_table/file/mod.rs

```diff
@@ -207,7 +209,10 @@ pub mod tests {
         let (table, _dir) = test_table();
 
         let re = |s, k| {
-            let regex_str = format!(r".*\.tmp.*{}{}{}{}{}{}\.json", MAIN_SEPARATOR, MAIN_SEPARATOR, s, MAIN_SEPARATOR, MAIN_SEPARATOR, k);
+            let regex_str = format!(
+                r".*\.tmp.*{}{}{}{}{}{}\.json",
+                MAIN_SEPARATOR, MAIN_SEPARATOR, s, MAIN_SEPARATOR, MAIN_SEPARATOR, k
```

Comment:
Shouldn't there be a backlash in front of the separators? Because, what does this do?
```ruby
/.*\.tmp.*//something//something/
```

This makes sense:
```ruby
/.*\.tmp.*\\something\\something/
/.*\.tmp.*\/something\/something/
```